humboldt-state-university-environmental-resources-engineering.csl

```diff
@@ -0,0 +1,181 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>Humboldt State University-Environmental Resources Engineering</title>
```

Comment:
This needs to be "Humboldt State University - Environmental Resources Engineering", with spaces around the hyphen.